test/integration/suites/upgrade/07-check-workload-svid

```diff
@@ -1,22 +1,15 @@
 #!/bin/bash
 
-set -o pipefail
-
-NUMCHECKS=15
-CHECKINTERVAL=3
-
-# Keep dumping the SVID until we've detected a rotation
-for ((i=1;i<=NUMCHECKS;i++)); do
-    log-info "checking X509-SVID after agent upgrade ($i of $NUMCHECKS)..."
-    docker-compose exec new-spire-agent \
-        /opt/spire/bin/spire-agent api fetch x509 \
-        -write /opt/test/after-agent-upgrade || fail-now "SVID check failed"
-    if ! cmp --silent svids/after-server-upgrade/svid.0.pem svids/after-agent-upgrade/svid.0.pem; then
-        # SVID has rotated
-        touch success
-        exit 0
-    fi
-    sleep "${CHECKINTERVAL}"
-done
-
-log-err "Timed out waiting for the SVID to rotate after upgrading the agent"
+log-info "checking X509-SVID after agent upgrade ($i of $NUMCHECKS)..."
```

Comment:
is $i and $NUMCHECKS still a thing with these latest changes?

Reply:
Whoops!